CONTRIBUTORS

```diff
@@ -1,2 +0,0 @@
-wagerlabs (Joel Reymont)		inspired and contributed some poll code
```

Comment:
I would not remove contributors. They did contribute to the code.

Reply:
This file has not been maintained for years, so either the list is available through Git or some other means or it is maintained but have a file crediting some people but not other people is not helpful at all.

Reply:
I've moved these two to the `README.md`, I think it should be an acceptable compromise.